### PR TITLE
Master release sync

### PR DESCRIPTION
**Pull request by udaydenduluri33**
Fix the conflicts between release branch and Master

The features contained with the toolkit does introduce a breaking change(RHF) although it has passed ALL service regression testing, its general awareness to all.
```
[x] Yes
[ ] No
```

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -29,21 +29,50 @@
 
 ### Version 3.1.2-missing-ftpa-tab-beta
 **EUI-3825** Fix for missing FTPA tab contents.
+### Version 4.0.0
```
Comment (paul-graham) [inline RELEASE-NOTES.md:32]: Why is `Version 4.0.0` below a load of `Version 3.x.x` in here? Shouldn't `4.0.0` be at the top?

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by kuda1992 — DISMISSED** [review]

**Review by paul-graham — CHANGES_REQUESTED** [review]
`RELEASE-NOTES.md` need reordering and I'm pretty sure it's introducing a breaking change seeing as `master` is a major version higher than `Release`.

File: RELEASE-NOTES.md (hunk 1 above)
Comment (udaydenduluri33) [reply RELEASE-NOTES.md:32]: Changed now.

**Review by kuda1992 — APPROVED** [review]

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by paul-graham — APPROVED** [review]

**Pull request merged**